packages/medusa/src/api/admin/exchanges/[id]/cancel/route.ts

```diff
@@ -4,10 +4,11 @@ import {
   MedusaResponse,
 } from "../../../../../types/routing"
 import { AdminPostCancelExchangeReqSchemaType } from "../../validators"
+import { HttpTypes } from "@medusajs/types"
 
 export const POST = async (
   req: AuthenticatedMedusaRequest<AdminPostCancelExchangeReqSchemaType>,
-  res: MedusaResponse
+  res: MedusaResponse<HttpTypes.AdminExchangeResponse>
```

Comment:
@shahednasser at the moment the Claim's cancel and Exchange's cancel workflows are returning void.
You can force cast the res.json here, but I will add later a remoteQuery call to return the correct type after the workflow is executed